monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
-=============================
-* Good enough uptime (a third party service to monitor the servers can be used to provide statistics)
-* Sybil Attack resistance (i.e. no one should be able to announce an unlimited number of (faulty) services)
-* Some degree of redundancy (ability to register balance proof with multiple competing monitoring services)
-* A stable and fast ethereum node connection (channel update transactions should be propagated ASAP as there is competition among the monitoring services)
-* If MS registry is used, a deposit will be required for registration
+Summary
+=======
+*   Monitoring Service (MS) listens to blinded Balance Proofs (BP) and proposed service fees in a public Matrix Room
+*   Any pre-registered MS can decide to monitor a channel and store corresponding BPs
+*   Then, whenever a channel is closed by calling `closeChannel` the MS will call `updateNonClosingBalanceProof` with
+    the submitted BP by its client
+*   Service fees are paid in RDN, there is no free tier
+*   In the short term we go for a simple design which will allow us to reach the Ithaca milestone earlier
+*   In the long-term we see the PISA approach as more economically viable and user friendly, but this design requires
+    additional features and can be developed independently after Ithaca
 
 Usual scenario
 ==============
 
-The Raiden node that belongs to Alice is going offline and Alice wants to be protected against having her channels closed by Bob with an incorrect :term:`balance proof`.
+The Raiden node that belongs to Alice is going offline and Alice wants to be protected against having her channels
+closed by Bob with an incorrect `balance proof`.
 
-1) Alice broadcasts the balance proof by sending a message to a public chat room.
-2) Monitoring services decide if the fee is worth it and picks the balance proof up.
-3) Alice now goes offline.
-4) Bob sends an on-chain transaction in attempt to use an earlier balance proof that is in his favor.
-5) Some of the monitoring servers detect that an incorrect :term:`BP` is being used. They can update the channel closing state as long as the :term:`Challenge Period` is not over.
-6) After the Challenge period expires, the channel can be settled with a balance that Alice expects to be correct.
+1)  Whenever Alice wants to get her channel monitored, Alice publishes her Monitor Request - blinded balance proofs -
+    into a public chat room
+2)  Updates can be sent whenever the state or proposed fee changes
+3)  MSs can pick up these messages and then listen to closeChannel events regarding this particular channel
+4)  If so, whenever the channel is closed and before the settlement period has ended, the MS will call the
+    `updateNonClosingBalanceProof` function with the provided information. This way the settleChannel function that
+    calculates the token distribution can only be executed submitting the corresponding balance values of the hashed
+    balances provided before
```

Comment:
Pls add that the client gets time to react first.

Reply:
done